pipeline/authn/authenticator_bearer_token_test.go

```diff
@@ -96,7 +96,7 @@ func TestAuthenticatorBearerToken(t *testing.T) {
 					w.WriteHeader(200)
 					w.Write([]byte(`{"sub": "123"}`))
 				},
-				config:    []byte(`{"preserve_query": true}`),
+				config:    []byte(`{"preserve_query": true, "forward_http_headers": ["Authorization"]}`),
```

Comment:
Just a question, i see here we are adding `forward_http_headers` for `Authorization` in many of these tests. Aren't they included automatically now?

Reply:
Yep. I wrote tests first and later I decided to implement some backward compatibility. I can remove it from the tests if you want.

Reply:
yeah i think it makes sense to remove it. otherwise it will just confuse in future.

We should just add a test though to make sure these headers are always present in these authentictors (explicitly even though the other tests infer it).

Reply:
Fixed it :)